src/bdr_solrizer/solrdocbuilder.py

```diff
@@ -61,24 +62,32 @@ def _process_extracted_text(data, content_type):
 
 class StorageObject:
 
-    def __init__(self, pid):
+    def __init__(self, pid, use_object_cache=False):
         self.pid = pid
         self._active_file_names = None
         self._active_file_profiles = None
-        self.files_response = self._get_files_response()
+        self.use_cache = use_object_cache
```

Comment:
suggestion: use self.use_object_cache here, so it stays obvious which cache we're talking about.